stateresolutionv2.go

```diff
@@ -17,10 +17,20 @@ package gomatrixserverlib
 import (
 	"container/heap"
 	"encoding/json"
+	"fmt"
 	"sort"
 	"strconv"
 )
 
+// Ordering represents how to sort a list of events, used primarily in ReverseTopologicalOrdering
+type Ordering int
```

Comment:
In the exported interface this might be a bit of a generic name, might be better as something like `TopologicalOrder`.